src/common/checksumalgorithms.h

```diff
@@ -29,15 +31,19 @@
 #endif
 namespace OCC {
 namespace CheckSums {
+    OCSYNC_EXPORT Q_NAMESPACE;
+
     enum class Algorithm {
         SHA3_256 = QCryptographicHash::Sha3_256,
         SHA256 = QCryptographicHash::Sha256,
         SHA1 = QCryptographicHash::Sha1,
         MD5 = QCryptographicHash::Md5,
         ADLER32 = 100,
         DUMMY_FOR_TESTS,
-        Error
+        NONE,
```

Comment:
Since this is the default value, it should probably be moved up to the top and assigned `0`.